### ISSUE TITLE
Question about the alignment-uniform figure.

### ISSUE THREAD

**Issue by Cosmos-Break** [princeton-nlp/SimCSE#52]
In the paper, it says ' We visualize checkpoints every 10 training steps and the arrows indicate the training direction. '
the calculation of alignment and uniformity is done on a training batch or some test set?

**Comment by gaotianyu1350** [comment]
For visualization, we take the whole STS-B dataset.

**Comment by Cosmos-Break** [comment]
thank you~
